docs/userguide/format_parse/messages/try-mf2.md

```diff
@@ -0,0 +1,366 @@
+       std::cout << strResult << std::endl;
+    }
+    ```
+
+1. Build your application and run it
+    ```sh
+    g++ hello_mf2.cpp -I$DESTDIR/usr/local/include -std=c++17 -L$DESTDIR/usr/local/lib -licuuc -licudata -licui18n
+
+    # if macOS
+    DYLD_LIBRARY_PATH=$DESTDIR/usr/local/lib ./a.out
+    # else if Linux
+    LD_LIBRARY_PATH=$DESTDIR/usr/local/lib ./a.out
+    # end
+    ```
+
+1. This will output
+    ```
+    Hello John, today is January 13, 2025!
+    ```
+
+## C++ Windows with Visual Studio
+
+### From command line
+
+Start the Visual Studio "x64 Native Tools Command Prompt for VS 20xx"
+
+1. Prepare a sandbox folder
+
+    ```cmd
+    set ICU_SANDBOX=%USERPROFILE%\hello_icu_mf2
+    md %ICU_SANDBOX%
+    cd %ICU_SANDBOX%
+    ```
+
+1. Build ICU4C (you only need to do this once):
+
+    ```cmd
+    git clone https://github.com/unicode-org/icu.git
+    
+    cd icu\icu4c
+    msbuild source/allinone/allinone.sln /p:Configuration=Release /p:Platform=x64 /p:SkipUWP=true
+    cd ..\..
+    
+    set DESTDIR=%ICU_SANDBOX%\icu_release
+    rd /q/s %DESTDIR%
+    md %DESTDIR%
+    xcopy icu\icu4c\include %DESTDIR%\include /E /V /I /Q /Y
+    xcopy icu\icu4c\bin64   %DESTDIR%\bin64   /E /V /I /Q /Y
+    xcopy icu\icu4c\lib64   %DESTDIR%\lib64   /E /V /I /Q /Y
+    ```
+
+1. Create a minimal C++ file here (we are in the `$ICU_SANDBOX` folder).
+Let's call it `hello_mf2.cpp`.
+The content is listed above (see the Linux section).
+
+1. Build your application and run it
+
+    ```sh
+    g++ hello_mf2.cpp -I$DESTDIR/usr/local/include -std=c++17 -L$DESTDIR/usr/local/lib -licuuc -licudata -licui18n
+    
+    # if macOS
+    DYLD_LIBRARY_PATH=$DESTDIR/usr/local/lib ./a.out
+    # else if Linux
+    LD_LIBRARY_PATH=$DESTDIR/usr/local/lib ./a.out
+    # end
+    ```
+
+1. This will output
+    ```
+    Hello John, today is January 13, 2025!
+    ```
+
+### From Visual Studio (UI)
+
+1. Prepare a sandbox folder. Let's call it `hello_icu_mf2`.
+
+1. Build ICU4C (you only need to do this once)
+
+    * Clone the ICU repository from
+      <https://github.com/unicode-org/icu> to the `hello_icu_mf2` folder.
+      We will end up with the folder `hello_icu_mf2\icu`.
+
+        > :point_right: **Note**: You can also downloaded a pre-build artifact from [GitHub releases](https://github.com/unicode-org/icu/releases/tag/release-76-1). But in that case the 
+        MessageFormat 2 implementation will be behind the spec.
+
+    * Start Visual Studio.
+    * Select _"Open a project or solution"_
+    * Open the `allinone.sln` solution from the `hello_icu_mf2\icu\icu4c\source\allinone` folder.
+    * Select the "Build" -- "Configuration Manager" menu
+    * Change the active solution to "Release" and "x64" (or another architecture, but you will have to be consistent everywhere after this)
+    * Select the "Build" -- "Build solution" menu
+    * Select the "File" -- "Close solution" menu
+
+1. Create a minimal C++ project in the `hello_icu_mf2` folder. 
```

Comment:
This has a trailing space.
That puts the "Let's call it `hello_mf2`." on a new line

But in the Linux section we don't do that, and "Let's call it..." is on the same line.

That's why I don't like "trailing spaces with meaning" :-)